packages/yoga/src/Dialog/web/Dialog.jsx

```diff
@@ -118,18 +108,17 @@ Dialog.propTypes = {
   dialogId: string,
   /** Control the dialog visibility. */
   isOpen: bool,
-  /** Hide the close button when onClose prop is defined. */
-  hideCloseButton: bool,
```

Comment:
Why did you remove `hideCloseButton` prop? It will break some implementations that are using it. 
Example, in partners portal.